### PR TITLE
FlxGame: make the filters array public

### PR DESCRIPTION
**Pull request by Sword352**
I remember doing something similar to FlxCamera, that would be nice to do the change to FlxGame as well

### CODE REVIEW

**Comment by Geokureli** [comment]
> home/runner/work/flixel/flixel/flixel/FlxGame.hx:148: characters 13-20 : Redefinition of variable filters in subclass is not allowed. Previously declared at openfl.display.DisplayObject

File: flixel/FlxGame.hx [hunk 1]
```diff
@@ -762,7 +766,7 @@ class FlxGame extends Sprite
 		FlxArrayUtil.clearArray(FlxG.swipes);
 		#end
 
-		filters = filtersEnabled ? _filters : null;
+		filters = filtersEnabled ? filters : null;
```
Comment (Geokureli) [inline flixel/FlxGame.hx:769]: This doesn't seem right

File: flixel/FlxGame.hx [hunk 2]
```diff
@@ -142,10 +142,13 @@ class FlxGame extends Sprite
 	 */
 	var _lostFocus:Bool = false;
 
-	/**
-	 * The filters array to be applied to the game.
-	 */
-	var _filters:Array<BitmapFilter>;
+        /**
+         * The filters array to be applied to the game.
+         */
```
Comment (Geokureli) [inline flixel/FlxGame.hx:147]: indented with 8 spaces, should use 1 tab

**Review by Geokureli — CHANGES_REQUESTED** [review]
FlxGame extends DisplayObject with has `public var filters(get, set):Array<BitmapFilter>;`
https://github.com/openfl/openfl/blob/develop/src/openfl/display/DisplayObject.hx#L462
the only solution I see is that we should keep _filters, and make the getter set _filters and instead of 
```hx
filters = filtersEnabled ? _filters : null;
```
it should do 
```hx
super.set_filters(filtersEnabled ? _filters : null);
```

**Comment by Sword352** [comment]
Did a few changes, this should be good now

**Comment by Geokureli** [comment]
still failing on flash with haxe 4.2.5

**Comment by Sword352** [comment]
Alright so there is a weird error with flash saying that flash.display.Sprite does not have a filters field, but when adding a conditional block for only compiling the override keyword to non-flash targets, it stil says it needs to be overrided on flash.
How can we solve that issue?

**Comment by Sword352** [comment]
Closing this, will rework on that later after i fix the flash issue

**Pull request closed without merging**